pkg/controller/direct/export.go

```diff
@@ -42,6 +43,9 @@ func Export(ctx context.Context, url string, config *config.ControllerConfig) (*
 
 		u, err := adapter.Export(ctx)
 		if err != nil {
+			if strings.Contains(err.Error(), "not supported") {
```

Comment:
Do we have to do this?  String matching against errors is pretty much an anti-pattern, only to be done if there's no choice

Reply:
I think I'll run back the `return nil, ErrNotImplemented` for now.